src/main/haskell/language-kore/src/Data/Kore/IndexedModule/IndexedModule.hs

```diff
@@ -412,7 +462,8 @@ indexModuleMetaSentence
         ( newIndexedModules
         , indexedModule
             { indexedModuleImports =
-                (attributes, importedModule) : indexedImports
+                (parseAttributes attributes, attributes, importedModule)
```

Comment:
Test? And you have the same issue in other similar places below.

Reply:
Since there are no tests at all for IndexedModule, I will add a separate task for that.